bot/tests/base_test.py

```diff
@@ -0,0 +1,117 @@
+import hashlib
+import logging
+from pathlib import Path
+import time
+from typing import (
+    List,
+    Optional,
+)
+
+from telethon.sync import TelegramClient
+from telethon.tl.custom.message import Message
+
+import bot.tests.messages as msg
+from bot.tests.settings import settings as s
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
mysle ze gdzie jak gdzie ale w testach to chyba chcesz miec jak najwiecej logow a nie obcinac debugi